jenkins/e2e-image/Dockerfile

```diff
@@ -52,4 +52,7 @@ ADD ["e2e-runner.sh", \
     "/workspace/"]
 RUN ["chmod", "+x", "/workspace/get-kube.sh", "/workspace/upload-to-gcs.sh", "/workspace/sh2ju.sh"]
 
+ADD ["configure-image.sh", "/tmp"]
+RUN ["/tmp/configure-image.sh"]
```

Comment:
(Sorry I've been out for a few weeks -- catching up on PRs now)

On the first point, I kept the logic in a separate script because it was a non-trivial number of lines and I wanted to keep the `Dockerfile` clean. If it's preferable to have the `Dockerfile` be self-contained, I can ditch `configure-image.sh`.

On the second point, I'm a little confused. The `kubekins-e2e` image is used for all of our `e2e` tests via `dockerized-e2e-runner.sh`, and I thought that it was used exclusively for that purpose. Are you saying that there are other consumers of `kubekins-e2e` aside from `e2e` tests, or are you suggesting that I create a separate image specifically for the `e2e` tests that require these new dependencies, and plumb through the ability to use arbitrary images in `dockerized-e2e-runner.sh`? I'm trying to figure out what you're trying to save from bloat: other `e2e` tests, or other things I don't know about.